src/components/Tooltip/__tests__/__snapshots__/tooltip-snapshot-tests.jest.js.snap

```diff
@@ -28,14 +28,14 @@ Array [
       className="monday-style-dialog-content-component top edge-center has-tooltip"
     >
       <div
-        className="monday-style-tooltip monday-style-tooltip-dark padding-size-undefined"
+        className="mondayStyleTooltip monday-style-tooltip mondayStyleTooltipDark monday-style-tooltip-dark paddingSizeUndefined padding-size-undefined"
```

Comment:
Is this an issue with jest&css module integration or a bug in the implementation?